source/application/BioXAS/BioXASImagingAppController.cpp

```diff
@@ -21,6 +21,7 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "BioXASImagingAppController.h"
 
+#include "beamline/CLS/CLSBeamlines.h"
```

Comment:
I will rename the Names. Unfortunately, we can't use a function to get the beamlineName now as you suggested since out Beamline IDs are NOT unique. For example, 5 could be either REIX beamline or IDEAS beamline.